Dockerfile.windows

```
@@ -248,7 +248,7 @@ RUN `
   Remove-Item C:\gitsetup.zip; `
   `
   Write-Host INFO: Creating source directory...; `
-  New-Item -ItemType Directory -Path C:\go\src\github.com\docker\docker | Out-Null; `
+  New-Item -ItemType Directory -Path C:\gopath\src\github.com\docker\docker | Out-Null; `
```

nit; this could use the `GOPATH` env-var, as I did in https://github.com/moby/moby/pull/38410;

```suggestion
  New-Item -ItemType Directory -Path ${GOPATH}\src\github.com\docker\docker | Out-Null; `
```